### PR TITLE
Crn 1344 add working groups and interest group to user profile dropdown

### PR DESCRIPTION
**Pull request by oratiu**

### CODE REVIEW

File: apps/crn-frontend/src/AuthenticatedApp.tsx [hunk 1]
```diff
@@ -79,6 +79,22 @@ const AuthenticatedApp: FC<Record<string, never>> = () => {
...
+          workingGroups={user.workingGroups.map(
+            ({ id, name = '', active }) => ({
+              name,
+              active,
+              href: network({})
+                .workingGroups({})
+                .workingGroup({ workingGroupId: id }).$,
+            }),
+          )}
+          interestGroups={(user.interestGroups || []).map(
```
Comment (oratiu) [inline apps/crn-frontend/src/AuthenticatedApp.tsx:91]: remove the || []

File: packages/react-components/src/molecules/UserNavigationAssociationSection.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+});
+
+const headlineStyle = css({
+  padding: `${12 / perRem}em ${12 / perRem}em`,
+  display: 'block',
+});
+
+const listStyle = css({
+  a: {
+    paddingTop: `${12 / perRem}em`,
```
Comment (oratiu) [inline packages/react-components/src/molecules/UserNavigationAssociationSection.tsx:19]: use one padding statement

File: packages/react-components/src/molecules/UserNavigationAssociationSection.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import { css } from '@emotion/react';
+import { Divider, Headline5, NavigationLink } from '../atoms';
+import { InterestGroupsIcon, TeamIcon, WorkingGroupsIcon } from '../icons';
+import { perRem } from '../pixels';
+
+const dividerStyle = css({
+  display: 'block',
+  marginLeft: `${12 / perRem}em`,
```
Comment (oratiu) [inline packages/react-components/src/molecules/UserNavigationAssociationSection.tsx:8]: use one margin statement

File: packages/react-components/src/organisms/UserNavigation.tsx [hunk 4]
```diff
@@ -34,6 +35,12 @@ const listStyles = css({
   padding: 0,
 });
 
+const dividerStyle = css({
+  display: 'block',
+  marginLeft: `${12 / perRem}em`,
```
Comment (oratiu) [inline packages/react-components/src/organisms/UserNavigation.tsx:40]: one margin statement ?

File: packages/react-components/src/molecules/__tests__/UserNavigationAssociationSection.test.tsx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+    />,
+  );
+
+  expect(getByText('group 1')).toBeVisible();
+  expect(getByText('MY INTEREST GROUPS')).toBeVisible();
+  expect(getByRole('link', { name: /interest group/i })).toHaveAttribute(
+    'href',
+    expect.stringMatching('/abc'),
+  );
+});
```
Comment (Riccardo-Brown) [inline packages/react-components/src/molecules/__tests__/UserNavigationAssociationSection.test.tsx:55]: Looks like the same test

**Review by gabiayako — COMMENTED** [review]
Hey, I've noticed that when the names are too big, for mobile it makes the screen to stretch more than the size of the screen, and it affects other screens like the dashboard page.


https://github.com/yldio/asap-hub/assets/16595804/e6960ba2-ce22-4538-8d11-d8e69b7584a6

I think if you add a `maxWidth: '100vw'` in the `menuStyles` in `packages/react-components/src/templates/Layout.tsx` this should be fixed 😉

**Review by kmaid — APPROVED** [review]

**Pull request merged**